pkg/macvlan/macvlan_test.go

```diff
@@ -0,0 +1,134 @@
+package macvlan
+
+import (
+	"fmt"
+	"testing"
+
+	egresstest "github.com/openshift/egress-router-cni/pkg/testing"
+	netlink_mocks "github.com/openshift/egress-router-cni/pkg/testing/mocks/github.com/vishvananda/netlink"
+	util "github.com/openshift/egress-router-cni/pkg/util"
+	util_mocks "github.com/openshift/egress-router-cni/pkg/util/mocks"
```

Comment:
why not just `mocks`?

Reply:
just to be consistent with the path `/pkg/util/mocks' but I'll change that if you feel it's more readable